src/api/store/event.py

```diff
@@ -54,70 +61,175 @@ def copy_event(self, context: Context, args) -> (dict, MassEnergizeAPIError):
       capture_message(str(e), level="error")
       return None, CustomMassenergizeError(e)
 
+  def list_recurring_event_exceptions(self, context: Context, args) -> (list, MassEnergizeAPIError):
+    community_id = args.pop("community_id", None)
+    subdomain = args.pop("subdomain", None)
+    user_id = args.pop("user_id", None)
+    
+    if community_id:
+      #TODO: also account for communities who are added as invited_communities
+      query =Q(community__id=community_id)
+      events = Event.objects.select_related('image', 'community').prefetch_related('tags', 'invited_communities').filter(query)
+      
+    elif subdomain:
+      query =  Q(community__subdomain=subdomain)
+      events = Event.objects.select_related('image', 'community').prefetch_related('tags', 'invited_communities').filter(query)
+      
+    elif user_id:
+      events = EventAttendee.objects.filter(attendee=user_id)
+      
+    else:
+      events = []
+    
+    exceptions = []
+    for event in events.all():
+      e = RecurringEventException.objects.filter(event=event).first()
+      if e:
+        exceptions.append(event.id)
+    
+    return exceptions, None
+
 
   def list_events(self, context: Context, args) -> (list, MassEnergizeAPIError):
     community_id = args.pop("community_id", None)
     subdomain = args.pop("subdomain", None)
     user_id = args.pop("user_id", None)
-
+    
     if community_id:
       #TODO: also account for communities who are added as invited_communities
       query =Q(community__id=community_id)
       events = Event.objects.select_related('image', 'community').prefetch_related('tags', 'invited_communities').filter(query)
+      
     elif subdomain:
       query =  Q(community__subdomain=subdomain)
       events = Event.objects.select_related('image', 'community').prefetch_related('tags', 'invited_communities').filter(query)
-
+      
     elif user_id:
       events = EventAttendee.objects.filter(attendee=user_id)
+      
     else:
       events = []
     
     if not context.is_sandbox and events:
       events = events.filter(is_published=True)
-  
+
     return events, None
 
 
   def create_event(self, context: Context, args) -> (dict, MassEnergizeAPIError):
+    
     try:
       image = args.pop('image', None)
       tags = args.pop('tags', [])
       community = args.pop("community_id", None)
-
+      is_recurring  = args.pop('is_recurring', False)
```

Comment:
looks like if "is_recurring" not passed in, and interpreted as False (not "false" or "") that recurring will be True